backend/index.js

```diff
@@ -19,4 +20,36 @@ app.get("/", (req, res) => {
   res.send("Hello world!");
 });
 
+// #1 - get all events
+app.get("/api/events", async (req, res) => {
+  try {
+    const events = await Event.find({});
+    res.send(events);
+  } catch (error) {
+    res.status(500).send(error.message);
+    console.log(`error is ${error.message}`);
+  }
+});
+
+// #2 - add an event
+app.post("/api/recipe", async (req, res) => {
+  try {
+    const { title, Location, start, end, slots, notes, Volunteers } = req.body;
+    let event = new Event({
+      title,
+      Location,
```

Comment:
I didn't catch Location and Volunteers were uppercase! Could you possible change the model so those are lowercase?